plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/AlteredFiles.java

```diff
@@ -47,7 +48,7 @@ public AlteredFiles(Project project, String diff) {
         alteredFilesStatuses = new LinkedHashMap<>();
         if (!isNullOrEmpty(diff)) {
             for (String item : diff.split("\n")) {
-                if (item.length() < 3 || item.charAt(1) != ' ') {
+                if (item.length() < 3 || item.charAt(1) != '\t') {
                     throw new IllegalArgumentException("Invalid git diff format.");
```

Comment:
If exception is thrown then it has to be caught properly. 
I am thinking about showing error with an appropriate message and skipping invalid `item`